apps/wallet-ui/src/app/[ecosystem]/(ui)/wallet/[address]/page.tsx

```diff
@@ -0,0 +1,35 @@
+import { getErc20Tokens } from "@/lib/assets/erc20";
+import { getErc721Tokens } from "@/lib/assets/erc721";
+import { SIMPLEHASH_SUPPORTED_CHAIN_IDS } from "@/util/simplehash";
+import { isAddress } from "thirdweb";
+import { getNativeBalances } from "../../../../../lib/assets/native";
+
+export default async function Page({
+  params: { address },
```

Comment:
hm, driving this from URL is a bit odd, it means that anyone can look at anyone's wallet contents via this page -- technically maybe fine but not what I would've expected

Reply:
my expecation here would be:

1. I connect wallet
2. I sign in (backend now has a cookie to work with)
3. the page here is driven by the address on the cookie